### ISSUE TITLE
suppress inv type messages about masternode stuff

### ISSUE THREAD

**Issue by chargr** [dashpay/p2pool-dash#18]
I think something like this should be safe to cut down on some logging.

```
diff --git a/p2pool/dash/p2p.py b/p2pool/dash/p2p.py
index eecbb54..8a04e12 100644
--- a/p2pool/dash/p2p.py
+++ b/p2pool/dash/p2p.py
@@ -76,7 +76,8 @@ class Protocol(p2protocol.Protocol):
             elif inv['type'] == 'block':
                 self.factory.new_block.happened(inv['hash'])
             else:
-                print 'Unneeded inv type', inv
+                if p2pool.DEBUG:
+                    print 'Unneeded inv type', inv
```

**Comment by poiuty** [comment]
yes, we can add it

**Comment by thelazier** [comment]
PR #24 already have this changes :)

**Comment by UdjinM6** [comment]
original issue was fixed in #19 , closing..
